Project/package.json

```diff
@@ -3,32 +3,32 @@
   "version": "0.3.0",
   "private": true,
   "dependencies": {
-    "@fluentui/react": "^7.137.1",
-    "react": "^16.13.0",
-    "react-dom": "^16.13.0",
-    "@azure/communication-common": "1.0.0-beta.3",
-    "@azure/communication-administration": "1.0.0-beta.3",
-    "@azure/communication-calling": "1.0.0-beta.3",
-    "@azure/logger": "1.0.0"
+    "@azure/communication-administration": "1.0.0-beta.4",
+    "@azure/communication-calling": "1.0.0-beta.4",
+    "@azure/communication-common": "1.0.0-beta.4",
+    "@azure/logger": "1.0.1",
```

Comment:
No need to specify these any more. Calling package lists these as dependencies in its packagejson

Reply:
Do not add common nor logger packages.

Please rm -rf node_modules, then run npm install + build and test to make sure all works fine.

Reply:
removed node_modules and npm installed, had no issue
tested and looks good